lib/authenticate_with_biometrics/view/biometric_button.dart

```diff
@@ -20,7 +20,7 @@ class BiometricButton extends StatelessWidget {
             .read<BiometricAuthBloc>()
             .add(const BiometricAuthEvent.checkSupport());
         return Card(
-          color:  Colors.grey,
+          color: Colors.grey,
```

Comment:
This is the right formatting but again it's not part of your changes.
It'll be good to remove to space as well. We want just the files that contain your implementation to be a part of this PR